Habinezajanvier/Capstone
# #16 edit profile

#### What does this PR do?
Edit admin profiles
#### Description of Task to be completed?
- Getting user Infos from HTML
- Displaying Infos on the edit form
- Submit edited data to the firebase DB
#### How should this be manually tested?
- Clone this repo `git clone https://github.com/Habinezajanvier/my-brand-project.git`
- Switch the directory `cd my-brand-project`
- Checkout the branch `git checkout ft-edit-profile-#16`
- Using your preferred browser start `index.html` found in `/admin/` folder
- Click on **Account** link found in left-side nav-bar
- Click on the edit profile button
- Edit your username and submit
> You should see edit username
#### Any background context you want to provide?
- If the page redirects you to login, use **`admin@email.com`** and **admin123** as credentials
#### What are the relevant pivotal tracker stories?
Click [here](https://trello.com/c/g3Y5C7OU/16-edit-profile-page) for relevant trello story
